graphics/jasper/Portfile

```diff
@@ -4,7 +4,8 @@ PortSystem      1.0
 PortGroup       cmake 1.0
```

Comment:
if you change this to the `cmake 1.1` PG, you can remove the line `cmake.out_of_source yes` below

Reply:
Done